plugins/provisioners/docker/provisioner.rb

```diff
@@ -19,7 +19,14 @@ def provision
         @logger = Log4r::Logger.new("vagrant::provisioners::docker")
 
         @logger.info("Checking for Docker installation...")
-        @installer.ensure_installed
+        if @installer.ensure_installed
```

Comment:
This conditional should also include a check to confirm a post install provisioner has been defined.